conf/machine/ls1043ardb.conf

```diff
@@ -22,7 +22,10 @@ UBOOT_CONFIG ??= "tfa-secure-boot tfa"
 UBOOT_CONFIG[tfa] = "ls1043ardb_tfa_defconfig,,u-boot-dtb.bin"
 UBOOT_CONFIG[tfa-secure-boot] = "ls1043ardb_tfa_SECURE_BOOT_defconfig,,u-boot-dtb.bin"
 
-KERNEL_DEVICETREE ?= "freescale/fsl-ls1043a-rdb-sdk.dtb freescale/fsl-ls1043a-qds.dtb freescale/fsl-ls1043a-qds-sdk.dtb"
```

Comment:
Please don't. For NXP specific one, please use:

```
KERNEL_DEVICETREE_append_use-nxp-bsp = "  ... "
```

Reply:
so qoriq needs to set similar var as IMX_DEFAULT_BSP. can we rename IMX_DEFAULT_BSP to be more generic and move its definition from imx-base.inc? or define a new QORIQ_DEFAULT_BSP?

Reply:
Define a new one.

Reply:
define QORIQ_DEFAULT_BSP and use KERNEL_DEVICETREE_append_use-nxp-bsp. PR is updated.